py/setup.py

```diff
@@ -69,6 +69,10 @@ def delete_scratchpad():
         rust_path = ".."
         scratchpad = None
 
+    # if the lib already built we replace the command
+    if os.environ.get("SKIP_RELAY_LIB_BUILD") is not None:
```

Comment:
Same here with the truthy/falsy values.

Reply:
same as in https://github.com/getsentry/relay/pull/1438#discussion_r959797823